templates/ajax.qtpl

```diff
@@ -28,6 +28,7 @@ Various AJAX-rendered HTML
 		{%= renderTags(org[common.Series], query) %}
 		{%= renderTags(org[common.Author], query) %}
 		{%= renderTags(org[common.Rating], query) %}
+		{%= renderTags(org[common.Name], query) %}
```

Comment:
Will need to render the name (if any) here instead. Probably at the top of the list.